### PR TITLE
Replace string formatting with f strings.

### PR DESCRIPTION
**Pull request by purple4reina**
### What does this PR do?

Replaces string formatting `"my {}".format("string")` with f strings `f"my {'string'}"`.

### Types of Changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [ ] This PR passes the integration tests (ask a Datadog member to run the tests)

### CODE REVIEW

File: datadog_lambda/cold_start.py [hunk 1]
```diff
@@ -50,12 +50,16 @@ def is_new_sandbox():
 
 def get_cold_start_tag():
     """Returns the cold start tag to be used in metrics"""
-    return "cold_start:{}".format(str(is_cold_start()).lower())
+    return "cold_start:true" if is_cold_start() else "cold_start:false"
```
Comment (duncanista) [inline datadog_lambda/cold_start.py:53]: Can't we just?
```suggestion
    return f"cold_start:{is_cold_start()}"
```

File: datadog_lambda/cold_start.py [hunk 2]
```diff
@@ -50,12 +50,16 @@ def is_new_sandbox():
...
 
 
 def get_proactive_init_tag():
     """Returns the proactive init tag to be used in metrics"""
-    return "proactive_initialization:{}".format(str(is_proactive_init()).lower())
+    return (
+        "proactive_initialization:true"
+        if is_proactive_init()
```
Comment (duncanista) [inline datadog_lambda/cold_start.py:60]: same as the cold start one?

**Review by astuyve — APPROVED** [review]

File: datadog_lambda/cold_start.py (hunk 1 above)
Comment (purple4reina) [reply datadog_lambda/cold_start.py:53]: Sure, but why do all that work when we already know what the tag will be?

Using python's `timeit` module:

```bash
$ python -m timeit -s 'from datadog_lambda.cold_start import is_cold_start' '"cold_start:true" if is_cold_start() else "cold_start:false"'
10000000 loops, best of 5: 25.2 nsec per loop
$ python -m timeit -s 'from datadog_lambda.cold_start import is_cold_start' 'f"cold_start{is_cold_start()}"'
5000000 loops, best of 5: 86.4 nsec per loop

Comment (purple4reina) [reply datadog_lambda/cold_start.py:53]: Oh, I found an even faster way!!

```python
return "cold_start:true" if _cold_start else "cold_start:false"
```

Cuts it down to `24.7 nsec per loop` from `40.9 nsec per loop`!

Comment (duncanista) [reply datadog_lambda/cold_start.py]: sounds good, thanks for the explanation

**Pull request merged**